### PR TITLE
os/bluestore: tiny tracepoints improvement

### PR DESCRIPTION
**Pull request by aclamk**
Fixed reversed (-) latency, synchronized all 'elapsed' times to seconds.

### CODE REVIEW

File: src/os/bluestore/BlueStore.cc [hunk 1]
```diff
@@ -11724,7 +11724,7 @@ void BlueStore::_kv_sync_thread()
       int deferred_size = deferred_stable.size();
 
 #if defined(WITH_LTTNG)
-      double sync_latency = ceph::to_seconds<double>(sync_start - mono_clock::now());
+      utime_t sync_latency = ceph_clock_now() - sync_start;
```
Comment (tchaikov) [inline src/os/bluestore/BlueStore.cc]: i think `mono_clock` is better in this case. as we are calculating the latency here, so the wall-clock time is not really necessary. BTW, the suggestion is made based on readability not on performance. their performances are identical.

Comment (aclamk) [reply src/os/bluestore/BlueStore.cc]: You think I should change all latency calculations in tracepoints to `mono_clock` ?

**Comment by athanatos** [comment]
I'd thought that we were trying to use the newer C++ time types in new code?

File: src/os/bluestore/BlueStore.cc (hunk 1 above)
Comment (aclamk) [reply src/os/bluestore/BlueStore.cc]: @tchaikov Converted tracepoint related time accesses to `mono_clock`, and `duration`.

Comment (tchaikov) [reply src/os/bluestore/BlueStore.cc]: @aclamk sorry for the latency! i didn't think you had to do this in this PR. but i am all for changing the latencies to `mono_clock`!

thanks!

**Pull request merged**